tests/CMakeLists.txt

```diff
@@ -106,13 +106,19 @@ IF(UNIX)
   ADD_EXECUTABLE(test_abstract_interface tools/test_abstract_interface.cpp)
   TARGET_LINK_LIBRARIES(test_abstract_interface
     pluginabstract ${CMAKE_DL_LIBS} ${Boost_LIBRARIES})
+  TARGET_INCLUDE_DIRECTORIES(test_abstract_interface
+    SYSTEM PUBLIC ${Boost_INCLUDE_DIRS})
+
 ENDIF(UNIX)
 
 FOREACH(path ${tests})
   GET_FILENAME_COMPONENT(test ${path} NAME)
   ADD_UNIT_TEST(${test} ${path}.cpp)
 
-  TARGET_LINK_LIBRARIES(${test} ${PROJECT_NAME} ${Boost_LIBRARIES}
+  TARGET_LINK_LIBRARIES(${test}
+    PUBLIC
+    ${PROJECT_NAME}
+    ${Boost_LIBRARIES}
```

Comment:
The point of adding the missing `PUBLIC` reported by @jmirabel was to avoid the need for `${Boost_LIBRARIES}` after `${PROJECT_NAME}`. @olivier-stasse : it still doesn't work without `${Boost_LIBRARIES}` ?

Reply:
Oh, no, we managed to avoid `${Boost_INCLUDE_DIRS}`. But in the same way, we should also be able to avoid the `${Boost_LIBRARIES}` :)

Reply:
Yes, we should also be able to do it. Except maybe for unit_test_framework.

Reply:
Thanks for your attention to this.
I removed all the unnecessary dependencies.
(I hope)